### DISCUSSION TITLE
При распаковке подпапки дважды создается родительская папка

### DISCUSSION THREAD

**Discussion by vasmedv** [elfmz/far2l#1119] [Q&A] [answered]
Os: Ububntu 20.04.3.
Far2l: 2.3.alpha build 20211023-9dfa589 Linux x86_64.
В Архиве лежит папка Языки, в ней папка English. 
Нажимаю Alt-F6. 
Рядом с архивом создается папка  Языки. 
В ней - вторая папка Языки.
И уже в ней только - папка English.
Это баг? 
Как это исправить?

**Comment by unxed** [comment]
Сходу не воспроизвелось. Образец архива бы

**Comment by vasmedv** [comment, marked as answer]
Попробовал на другом компьютере с Ubuntu. Там нормально распаковывается. Посмотрел - коммандные строки - zip:
там в распаковке не хватало -@%%R
